x/plans/keeper/migrations.go

```diff
@@ -22,3 +25,39 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 	}
 	return nil
 }
+
+func (m Migrator) Migrate3to4(ctx sdk.Context) error {
```

Comment:
1. The migration from 2 to 3 didn't happen yet, so let's combine this into the exiting/upcoming Migrate2to3().

Reply:
done